web/src/cypress/integration/alerts.ts

```diff
@@ -246,47 +246,132 @@ function testAlerts(screen: ScreenFormat) {
+      // Service Selection
+      cy.get('@dialog').contains('button', 'Next')
+
+      cy.get('@dialog')
+        .find('input[name=serviceSearch]')
+        .type(svc1.name)
 
-        cy.get('div[role=dialog]').as('dialog')
+      cy.get('@dialog')
+        .contains('span', svc1.name)
+        .click()
 
-        const summary = c.sentence({ words: 3 })
-        const details = c.word({ length: 10 })
+      cy.get('@dialog')
+        .find('input[name=serviceSearch]')
+        .clear()
 
-        cy.get('@dialog')
-          .find('input[name=summary]')
-          .type(summary)
-        cy.get('@dialog')
-          .find('input[name=service]')
-          .selectByLabel(svc.name)
+      cy.get('@dialog')
+        .find('[data-cy=service-chip-container]')
+        .contains('[data-cy=service-chip]', svc1.name)
+        .should('be.visible')
 
-        cy.get('@dialog')
-          .find('textarea[name=details]')
-          .type(details)
-        cy.get('@dialog')
-          .contains('button', 'Submit')
-          .click()
+      cy.get('@dialog')
+        .find('input[name=serviceSearch]')
+        .type(svc2.name)
 
-        // should be on details page
-        cy.get('*[data-cy=alert-summary]').should('contain', summary)
-        cy.get('*[data-cy=alert-details]').should('contain', details)
+      cy.get('@dialog')
+        .contains('span', svc2.name)
+        .click()
 
-        cy.go(-1)
+      cy.get('@dialog')
+        .find('[data-cy=service-chip-container]')
+        .contains('[data-cy=service-chip]', svc2.name)
+        .should('be.visible')
 
-        cy.pageSearch(summary)
-        cy.get('body').should('contain', svc.name)
-      })
+      cy.get('@dialog')
+        .find('[data-cy=service-chip-container]')
+        .contains('[data-cy=service-chip]', svc1.name)
+        .should('be.visible')
+
+      cy.get('@dialog').contains('label', 'Selected Services (2)')
+
+      // TODO test Add All button
+      // TODO test Filter button
+
+      cy.get('@dialog')
+        .contains('button', 'Next')
+        .click()
+
+      // Confirm
+      cy.get('@dialog')
+        .contains('span', svc1.name)
+        .should('be.visible')
+
+      cy.get('@dialog')
+        .contains('span', svc2.name)
+        .should('be.visible')
+
+      cy.get('@dialog')
+        .contains('[data-cy=service-chip]', svc1.name)
+        .should('be.visible')
+
+      cy.get('@dialog')
+        .contains('[data-cy=service-chip]', svc2.name)
+        .should('be.visible')
+
+      cy.get('@dialog')
+        .contains('button', 'Submit')
+        .click()
+
+      // Review
+      cy.get('@dialog')
+        .contains('button', 'Back')
+        .should('not.be.visible')
+
+      cy.get('@dialog')
+        .contains('button', 'Done')
+        .should('be.visible')
+
+      cy.get('@dialog')
+        .find('li')
+        .should('have.length', 2)
+
+      cy.get('@dialog')
+        .contains('button', 'Done')
+        .click()
+
+      //TODO test errors
```

Comment:
If we're going to test this we should do it -- or we can just keep it happy-path for this work.